routes.py

```diff
@@ -131,11 +131,28 @@ def regenerate_pat():
     return jsonify(result), status
 
 
-@app.route('/my-friends')
+@app.route('/my-friends', methods=['GET', 'POST'])
 def my_friends():
+    form = forms.UserFriendForm()
+
+    # TODO Properly handle the case case the user submits the form without being authenticated
+    if current_user.is_authenticated and form.validate_on_submit():
```

Comment:
Properly handle the case case the user submits the form without being authenticated (with proper error)